src/tactic/ui/tools/pipeline_wdg.py

```diff
@@ -5978,6 +6121,8 @@ def get_buttons_wdg(self, show_gear):
 
             server = TacticServerStub.get();
             spt.app_busy.show("Saving project-specific pipeline ["+group_name+"]",null);
+
+            console.log(node_kwargs);
```

Comment:
remove

Reply:
this console log is really useful for debugging